src/tilemaker/README.md

```diff
@@ -0,0 +1,49 @@
+# Tilemaker
+
+This container generates raster tiles using the results of an analysis job. It's run along with
+analysis jobs and uses the shapefiles exported by the analysis.
+
+The easiest way to run it locally is to load the `AnalysisJob` object you're interested in and
+call its `generate_tiles` method.  That won't run the tilemaker command but will print the right
+command to run to the logs, so you can copy it from there and run it.
+
+
+## Adding and Editing Styles
+
+The tiles are defined using Mapnik XML style files in the `styles` directory.
+Since the style XML files aren't templated, the path of the shapefile that's used as the
+data source is hard-coded to a file path in /data/.  The `run_tilemaker.sh` entrypoint script
+downloads the required shapefiles to the expected paths.
+
+For ease of editing and previewing, the styles are defined and maintained in a
+Mapbox Studio Classic project from which the XML style definitions are extracted by hand.
+
+Changes to the working files should be committed so that they're kept in sync with the actual
+tile styles being applied.
+
+### To open and edit the styles
+
+- Download the latest release of Mapbox Studio Classic from https://mapbox.s3.amazonaws.com/mapbox-studio/index.html (the [Mapbox download page](https://www.mapbox.com/mapbox-studio-classic/) links to an older, seemingly-broken version) and install it (for Linux,
+that just means unzipping it and running the `atom` command from the base directory).
+- Modify the hard-coded absolute paths in the following files to point to the actual locations
```

Comment:
I didn't have any issues doing this because I knew what to look for, but it may be helpful to include a sample file path that the user would be expected to change so they know what to look for. Then again maybe not, since the path currently there (`/home/local/...`) will probably accidentally change over time because the file is checked in. I think we can expect any developer looking at this to be able to figure out what to change based on the description provided. Maybe just change the wording to explicitly state "all", e.g. "Modify all hard-coded absolute paths in the following files..."?